ci/environment-unpinned.yml

```diff
@@ -0,0 +1,7 @@
+name: sliderule-ci
+channels:
+  - conda-forge
+dependencies:
+  - python
+  - pip
+  - pytest
```

Comment:
This will soon grab python 3.10 now that it is out. Also, the other ci environments get geopandas from conda-forge (geospatial packages installed from conda-forge tend to be more reliable compared to installed from pip). But wanted to also test the case where all dependencies (including geopandas) come from `pip install sliderule`